src/postcss-loader.js

```diff
@@ -185,20 +185,40 @@ export default {
 
     const cssVariableName = identifier('css', true)
     if (shouldExtract) {
-      output += `export default ${JSON.stringify(modulesExported[this.id])};`
+      // If the file is using css modules and the file exports any classes then
+      // the default export will be that mapping. If a css-modules file defines
+      // no classnames then the default export will not be present.
+      if (supportModules && modulesExported[this.id] !== undefined) {
+        output += `export default ${JSON.stringify(modulesExported[this.id])};`
```

Comment:
By only having a default export (in both extract and inject mode) when there is a classnames block then we remove the undefined variable blocks down in https://github.com/wardpeet/rollup-plugin-postcss/pull/1/files#diff-26e4d9ebc388064364a6cf68d73f91f8L1212-L1222